riff-raff/public/docs/riffraff/administration/auth.md

```diff
@@ -8,6 +8,19 @@ Authentication
 Authentication is handled by Google. This obtains the users full name and e-mail address and both of these are used to 
 provide an audit trail of deployments and other actions in the app.
 
+Google auth is implemented in two layers
+- Load Balancer layer: this component is defined in the main riff raff cloudformation template and uses [authenticate-oidc](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/listener-authenticate-users.html#configure-user-authentication). The template relies on a secret stored in SSM.
+- Application layer: this layer uses The Guardian's [Play Google Auth Module](https://github.com/guardian/play-googleauth) and is configured in the main [application.conf](https://github.com/guardian/riff-raff/blob/afb7e602e11acd7a07aae433c74be22976d8a7cd/riff-raff/conf/application.conf#L40-L41).
+
+To rotate the secrets, follow these steps:
+1. Create a new Client Secret in Riff Raff's Google Cloud project
+2. Patch the PROD and CODE application.conf to use the new secret
+3. Update the secret in AWS Secrets Manager used by the cloudformation template.
+4. Use riff raff to deploy itself, which will re-run the application startup routine while keeping the same instance alive
```

Comment:
Minor: as the Riff-Raff deployment is a little unusual it might be worth explicitly stating that this is sufficient to reload the config?

```suggestion
4. Use riff raff to deploy itself, which will re-run the application startup routine (including loading the updated `application.conf`) while keeping the same instance alive
```

Reply:
Yep, great shout!